server/feathers/src/app.js

```diff
@@ -61,9 +61,11 @@ app.get("/savedProducts", (req, res) => {
       ],
     })
     .then(user => {
+      res.send(200);
```

Comment:
this status code will be sent back automatically.  also, only one send per block